### PR TITLE
chore: remove unused code

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
 remove unused code 
1. chore resource_huaweicloud_oms_task_v1: use the unused func: getSmnInfo 
2. delete the scale test in dli queue basic test, (The queue must have executed sql to perform scale_out)

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/vpc' TESTARGS='-run=TestAccNetworkingSecGroupsDataSource'
--- PASS: TestAccNetworkingSecGroupsDataSource_basic (19.10s)
--- PASS: TestAccNetworkingSecGroupsDataSource_description (19.17s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpc       19.255s
```
```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccProvider'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run=TestAccProvider -timeout 360m -parallel 4
=== RUN   TestAccProvider_caCertFile
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       0.090s
```

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccBlockStorageV2Volume'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run=TestAccBlockStorageV2Volume -timeout 360m -parallel 4
=== RUN   TestAccBlockStorageV2Volume_basic
=== PAUSE TestAccBlockStorageV2Volume_basic
=== RUN   TestAccBlockStorageV2Volume_online_resize
=== PAUSE TestAccBlockStorageV2Volume_online_resize
=== RUN   TestAccBlockStorageV2Volume_image
=== PAUSE TestAccBlockStorageV2Volume_image
=== RUN   TestAccBlockStorageV2Volume_timeout
=== PAUSE TestAccBlockStorageV2Volume_timeout
=== CONT  TestAccBlockStorageV2Volume_basic
=== CONT  TestAccBlockStorageV2Volume_image
=== CONT  TestAccBlockStorageV2Volume_online_resize
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       0.081s
```

```
 make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccIecNetworkACLRuleResource'
=== RUN   TestAccIecNetworkACLRuleResource_basic
--- PASS: TestAccIecNetworkACLRuleResource_basic (32.41s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       32.598s
```
```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccIecSecurityGroupResource'
=== RUN   TestAccIecSecurityGroupResource_basic
--- PASS: TestAccIecSecurityGroupResource_basic (22.23s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       22.331s
```

```
 make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccNetworkingV2FloatingIP'
=== RUN   TestAccNetworkingV2FloatingIP_basic
=== RUN   TestAccNetworkingV2FloatingIP_fixedip_bind
--- SKIP: TestAccNetworkingV2FloatingIP_basic (0.00s)
--- SKIP: TestAccNetworkingV2FloatingIP_fixedip_bind (0.00s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       0.083s
```

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccNetworkingV2Port'
=== RUN   TestAccNetworkingV2PortDataSource_basic
=== PAUSE TestAccNetworkingV2PortDataSource_basic
=== RUN   TestAccNetworkingV2Port_basic
=== PAUSE TestAccNetworkingV2Port_basic
=== RUN   TestAccNetworkingV2Port_noip
=== PAUSE TestAccNetworkingV2Port_noip
=== RUN   TestAccNetworkingV2Port_timeout
=== PAUSE TestAccNetworkingV2Port_timeout
=== RUN   TestAccNetworkingV2Port_createExtraDHCPOpts
=== RUN   TestAccNetworkingV2Port_updateExtraDHCPOpts
--- PASS: TestAccNetworkingV2PortDataSource_basic (12.99s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       13.094s
```

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccComputeVolumeAttach'
=== RUN   TestAccComputeVolumeAttach_basic
=== RUN   TestAccComputeVolumeAttach_device
--- PASS: TestAccComputeVolumeAttach_device (200.70s)
--- PASS: TestAccComputeVolumeAttach_basic (205.64s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       205.779s
```

```
make testacc TEST='./huaweicloud/services/acceptance/dli' TESTARGS='-run=TestAccDliQueue'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dli -v -run=TestAccDliQueue -timeout 360m -parallel 4
=== RUN   TestAccDliQueue_basic
=== PAUSE TestAccDliQueue_basic
=== RUN   TestAccDliQueue_cidr
=== PAUSE TestAccDliQueue_cidr
=== CONT  TestAccDliQueue_basic
=== CONT  TestAccDliQueue_cidr
--- PASS: TestAccDliQueue_basic (260.11s)
--- PASS: TestAccDliQueue_cidr (268.46s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dli       268.542s
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_compute_volume_attach_test.go [hunk 1]
```diff
@@ -121,18 +121,6 @@ func testAccCheckComputeVolumeAttachExists(n string, va *block_devices.VolumeAtt
 	}
 }
 
-func testAccCheckComputeVolumeAttachDevice(
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_compute_volume_attach_test.go:124]: suggest to keep this function,we will refactor later.

File: huaweicloud/resource_huaweicloud_iec_network_acl_rule_test.go [hunk 2]
```diff
@@ -45,27 +45,6 @@ func TestAccIecNetworkACLRuleResource_basic(t *testing.T) {
 	})
 }
 
-func testAccCheckIecNetworkACLRuleDestroy(s *terraform.State) error {
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_iec_network_acl_rule_test.go:48]: ditto

File: huaweicloud/resource_huaweicloud_iec_security_group_test.go [hunk 3]
```diff
@@ -44,17 +44,6 @@ func TestAccIecSecurityGroupResource_basic(t *testing.T) {
 	})
 }
 
-func testAccCheckIecSecurityGroupV1RuleCount(group *groups.RespSecurityGroupEntity, count int) resource.TestCheckFunc {
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_iec_security_group_test.go:47]: ditto

File: huaweicloud/resource_huaweicloud_compute_volume_attach_test.go (hunk 1 above)
Comment (hmbig2) [reply huaweicloud/resource_huaweicloud_compute_volume_attach_test.go:124]: pls add when you need really

**Pull request closed without merging**